Geometry_Engine/Modify/Offset.cs

```diff
@@ -590,7 +512,7 @@ private static List<ICurve> ExtendToPoint(this ICurve curve, Point startPoint, P
 
         /***************************************************/
 
-        private static PolyCurve Fillet(this ICurve curve1, ICurve curve2, bool tangentExtensions, bool keepCurve1StartPoint, bool keepCurve2StartPoint, double tolerance = Tolerance.Distance)
+        private static PolyCurve Fillet(this ICurve curve1, ICurve curve2, bool tangentExtensions, bool keepCurve1StartPoint, bool keepCurve2StartPoint, double distTol = Tolerance.Distance, double angleTol = Tolerance.Angle)
```

Comment:
I have just noticed there is an identical method (literally copy-paste 🤯) in OffsetVariable.cs in Facade_Engine - maybe worth unifying before things go out of sync? Should be literally deletion of the one in Facade_Engine

Reply:
As discussed offline, will not touch for now given this is a private method. The method in Facade_Engine most likely should be moved over to Geometry_Engine, and if/when that is done, it can be removed.